README.md

```diff
@@ -300,23 +300,25 @@ switches.
 Achieving the highest possible benchmarking quality requires more care. First,
 none of Krun's debug or development switches must be used. Second, Krun needs to
 run in "reboot mode" where each process execution will be run after the machine
-has (automatically) rebooted. The recommended way to do this is via `cron(8)`
-and the `start_krun_from_cron` script supplied with Krun.
-
-As a regular user (who has access to the experiment and the Krun code), run
-`crontab -e` and add a line similar to the following:
+has (automatically) rebooted. The simplest way to do this is to have your init
+system invoke `scripts/run_krun_at_boot` via a rc.local(8) script.
 
+Your `/etc/rc.local` should look like this:
 ```
-@reboot /path/to/krun/scripts/start_krun_from_cron /path/to/your/config.krun
+#!/bin/sh
+sudo -u someuser /path/to/krun/scripts/run_krun_at_boot /path/to/your/config.krun
+exit 0
 ```
 
-Note that: the path to `Krun` must be an absolute paths for `cron` to be
-able to run it; and the config file must be an absolute path for `Krun` to
-be able to use it. Any arguments supplied after the config file path are
-passed to Krun unchanged.
+Make sure you replace the paths as appropriate and substitute `someuser` with
+the name of a normal unprivileged user that you wish to use to kick off Krun.
```

Comment:
Should this not be the `krun` user?

Reply:
No, that gets created and removed automatically. For real experiments I actually use a `kruninit` user account with just the experiments in.

Reply:
OK, cool